KOService.WebAPI/Infrastructure/AutoMapperConfiguration.cs

```diff
@@ -30,11 +30,16 @@ public static void Configure()
                    .ForMember(dest => dest.Status, opt => opt.MapFrom(src => src.GetStatus()));
 
                 cfg.CreateMap<Activity, ActivityDto>()
-                   .ForMember(dest => dest.MechanicName, opt => opt.MapFrom(src => $"{src.Mechanic.FirstName} {src.Mechanic.LastName}"))                
+                   .ForMember(dest => dest.MechanicName, opt => opt.MapFrom(src => $"{src.Mechanic.FirstName} {src.Mechanic.LastName}"))
+                   .ForMember(dest => dest.VehicleBrand, opt => opt.MapFrom(src => src.Repair.Vehicle.Type.Brand))
                    .ForMember(dest => dest.VehicleRegistrationNumbers, opt => opt.MapFrom(src => src.Repair.Vehicle.RegistrationNumbers))
-                   .ForMember(dest => dest.VehicleBrand, opt => opt.MapFrom(src => src.Repair.Vehicle.Type.Brand));
+                   .ForMember(dest => dest.Status, opt => opt.MapFrom(src => src.GetStatus()));
```

Comment:
Dlaczego tu dodałeś znowu to mapowanie do statusu? Coś ci nie działa bez tego?

Reply:
no nie działa bez tego

Reply:
i tak samo jest z tym filtrowaniem po statusach